scipy/io/wavfile.py

```diff
@@ -150,19 +151,19 @@ def _read_riff_chunk(fid):
 
     return fsize, is_big_endian
 
-# open a wave-file
-
 
 def read(filename, mmap=False):
     """
+    Open a WAV file
+
     Return the sample rate (in samples/sec) and data from a WAV file.
 
     Parameters
     ----------
     filename : string or open file handle
-        Input wav file.
+        Input wav file.  This can be an open file or a filename.
```

Comment:
I don't think we really need it either place -- seems clear enough from the enumerated types. WDYT?